doc/adding_lints.md

```diff
@@ -432,12 +432,12 @@ don't hesitate to ask on Discord, IRC or in the issue/PR.
 [diagnostics]: https://github.com/rust-lang/rust-clippy/blob/master/clippy_lints/src/utils/diagnostics.rs
 [utils]: https://github.com/rust-lang/rust-clippy/blob/master/clippy_lints/src/utils/mod.rs
 [ident]: https://doc.rust-lang.org/nightly/nightly-rustc/syntax/source_map/symbol/struct.Ident.html
-[span]: https://doc.rust-lang.org/nightly/nightly-rustc/syntax_pos/struct.Span.html
+[span]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_span/struct.Span.html
```

Comment:
The link points to wrong place at now, but it should work well once new nightly is landed.

Reply:
It returns correct place now.